SalesforceCommerceCloud/pwa-kit
# Update bump version script to include peerDependencies

# Description

The `pwa-kit-runtime` has the `pwa-kit-dev` as a peerDependency:

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/v2/packages/pwa-kit-runtime/package.json#L67-L69

The NPM bump-version script at the root of the monorepo doesn't update the monorepo packages included as a peerDependecies. As a result, we see this error when trying to run `npm ci` on some of the Circle CI jobs:

Circular dependency not satisfied "pwa-kit-dev@2.0.0-alpha.0" -> "pwa-kit-runtime" -> "pwa-kit-dev@^2.0.0-dev.4"

A way to fix this is to modify the bump-version.js to check for local packages that are listed a peerDepdency and update their versions

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Check out this branch
- Run bump-version script by `npm run bump-version -- 2.1.0-dev.1`
- Verify that all references of locals packages both dependencies and peerDependencies are all updated with 2.1.0-dev.1

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
